gluten-data/pom.xml

```diff
@@ -109,7 +109,7 @@
     <dependency>
       <groupId>org.apache.arrow</groupId>
       <artifactId>arrow-vector</artifactId>
-      <version>${arrow.version}</version>
+      <version>${arrow-gluten.version}</version>
       <exclusions>
         <exclusion>
           <groupId>io.netty</groupId>
```

Comment:
Where do the jars come from on CI? As I see only `arrow-dataset-gluten.jar` was built and uploaded for CI jobs.

![Uploading image.png…]()